eventhub/Module.md

```diff
@@ -1,540 +1,199 @@
-# Ballerina Azure Event Hubs Module
+## Overview
 
-Connects to Microsoft Azure Event Hubs using Ballerina.
+Ballerina Connector for Azure Event Hub provides access to the Azure Event Hub via Ballerina language to ingest, buffer, store, and process high-frequency data from any source in real-time to derive important business insights. 
```

Comment:
Let's rephrase this to "Azure Event Hub connector provides access to the Azure Event Hub via the Ballerina Language. This connector allows you to ingest, buffer, store, and process high-frequency data from any source in real-time to derive important business insights using the Azure Event Hub. 	 
  
It provides the capability to perform Event Hub service operations, Event Hub management operations, handle publisher policy operations."

Reply:
Resolved and amended the changes. Thank you for the suggestion.